### DISCUSSION TITLE
Github Actions not triggering for new tags

### DISCUSSION THREAD

**Discussion by gandarez** [community/community#24968] [Actions]
<p>I just realized that any new pushed tag while running under an Action won’t trigger push event.</p>
<p><strong>on_push.yml</strong> triggers when any push is done for master branch and creates a new tag.</p>
<pre><code class="lang-auto">on:
  push:
    branches:
      - "master"

jobs:  
  tag:
    name: Create Tag
    runs-on: ubuntu-latest
    steps:
      - name: "Checkout"
        uses: actions/checkout@v2
        with:
          fetch-depth: 0
      - name: "Create semver tag"
        id: semver-tag
        uses: ....
</code></pre>
<p><strong>on_tag.yml</strong> listen for tag creations and trigger another pipeline to do other stuff (like release the product). However it’s ony being triggered when pushing a new tag from outside Actions, for examploe, from my local machine.</p>
<pre><code class="lang-auto">on:
  push:
    tags:
      - '*'

jobs:
  release:
    name: Release
    runs-on: ubuntu-latest
    steps:
      - run: echo ${{ github.ref }}
</code></pre>

**Comment by airtower-luna** [comment, marked as answer]
<p>See the documentation on <a href="https://docs.github.com/en/actions/reference/authentication-in-a-workflow#using-the-github_token-in-a-workflow">Using the <code>GITHUB_TOKEN</code> in a workflow</a>:</p>
<blockquote>
<p>When you use the repository’s <code>GITHUB_TOKEN</code> to perform tasks on behalf of the GitHub Actions app, events triggered by the <code>GITHUB_TOKEN</code> will not create a new workflow run.</p>
</blockquote>
<p>If you want to push a tag from a workflow and have the push trigger another workflow you need to use a PAT for authentication instead, see <a href="https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token">Creating a personal access token</a>.</p>
